gh-133810: remove `http.server.CGIHTTPRequestHandler` and `--cgi` flag (#133811)
python/cpython: Misc/NEWS.d/next/Library/2025-05-10-11-04-47.gh-issue-133810.03WhnK.rst

> Remove :class:`!http.server.CGIHTTPRequestHandler` and ``--cgi`` flag to the

Reviewer: ```suggestion
Remove :class:`!http.server.CGIHTTPRequestHandler` and ``--cgi`` flag from the
```

:program:`python -m http.server` command-line interface. Those are

> deprecated since 3.13. Patch by Bénédikt Tran.

Author: ```suggestion
deprecated since Python 3.13. Patch by Bénédikt Tran.
```

> deprecated since Python 3.13. Patch by Bénédikt Tran.

Reviewer: Perhaps also?
```suggestion
:program:`python -m http.server` command-line interface. They were
deprecated in Python 3.13. Patch by Bénédikt Tran.
```
